ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +650,110 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_ocs(self):
+        """
+        The function Uninstalls the OCS operator from a openshift
+        cluster and removes all its settings and dependencies
+
+
+        Returns
+            Boolean: True if operation was successful
+
+        """
+        ocp_obj = ocp.OCP()
+        provisioners = constants.OCS_PROVISIONERS
+
+        # STEP 1:List the storage classes
+        sc_list = get_all_storageclass(all_ns=True)
```

Comment:
Storageclass does not belong to any namespace storageclass can be used in any namespace all_ns=True is not required here